test-descriptor/test-descriptor-schema.yml

```diff
@@ -0,0 +1,92 @@
+---
+$schema: "http://json-schema.org/draft-04/schema#"
+title: "Test Descriptor Schema"
+version: 0.9
+description: "The schema for 5GTANGO Test Descriptor"
+
+##
+## Some definitions used later on.
+##
+definitions:
+  testtype:
+     enum:
+       - "TTCN3"
+       - "Python"
+       - "moongen"
+
+##
+## The actual document description.
+##
+type: "object"
+properties:
+  descriptor_schema:
+    description: "Reference to the schema corresponding to the descriptor (e.g., URL or local path)."
+    type: "string"
+  vendor:
+    description: "The vendor id allows identification of a test descriptor uniquely."
+    type: "string"
+    pattern: "^[a-z0-9\\-_.]+$"
+  name:
+    description: "The name of the test description."
+    type: "string"
+    pattern: "^[a-z0-9\\-_.]+$"
+  version:
+    description: "The version of the test descriptor."
+    type: "string"
+    pattern: "^[0-9\\-_.]+$"
+  author:
+    description: "The person or organization that created the test descriptor."
+    type: "string"
+  description:
+    description: "A longer description of the test descriptor."
+    type: "string"
+  test_type:
+    description: "test type"
+    $ref: "#/definitions/testtype"
+  test_execution_file:
```

Comment:
Can be multiple files?

Reply:
hmm maybe test_resources is a better and more generic name?

Reply:
and yes can be multiple files